config/v1alpha1/types_cluster_monitoring.go

```diff
@@ -72,22 +74,33 @@ type ClusterMonitoringList struct {
 }
 
 // ClusterMonitoringSpec defines the desired state of Cluster Monitoring Operator
-// +required
 type ClusterMonitoringSpec struct {
```

Comment:
We don't want `{}` to be a valid value for spec, you should add a minimum properties
```suggestion
// +kubebuilder:validation:MinProperties:=1
type ClusterMonitoringSpec struct {
```